### PR TITLE
Improvements for the FAQ

### PR DESCRIPTION
**Pull request by codeZeilen**

### CODE REVIEW

File: FAQ/FAQ.tex [hunk 1]
```diff
@@ -41,44 +41,61 @@ \section{Getting started}
...
+Open the yellow button menu of the world (by clicking the yellow button in the world). 
+There you find the menu item \menu{show main docking bar}.
+Click it once to deactivate it completely, and then click it a second time to re-activate it.
+
+\begin{faq}
+Some code triggered a lot of debuggers and my whole screen is filled with them. 
+How can I get rid of them?
+\end{faq}
+\answer
+You can close all windows that look similar at once by selecting \menu{World docking bar \go Windows \go Bad Window \go Close all like this}.
```
Comment (LinqLover) [inline FAQ/FAQ.tex:68]: Just by interested, how is this different from Do > Utilities closeAllDebuggers? Is the latter redundant?

Comment (codeZeilen) [reply FAQ/FAQ.tex:68]: I don't think there is any difference in the actual functionality, except that `close all like this` is more general, and I wanted to convey this more general way of managing windows. 
(Also, I guess `closeAllDebuggers` might be more robust against very large numbers of debuggers, as the list of menus requires quite a long time to build up in this case)

Comment (LinqLover) [reply FAQ/FAQ.tex:68]: And `closeAllDebuggers` is better reachable using the keyboard only (Ctrl + 5, d, Enter). However. :-)

**Comment by codeZeilen** [comment]
... debugging latex without helpful log output is such fun ...

**Pull request merged**